config/initializers/faraday.rb

```diff
@@ -0,0 +1,2 @@
+require 'faraday'
+Faraday.default_adapter = :excon
```

Comment:
excon is just better! no, really it makes it easier to debug consistently since then I can set ENV['EXCON_DEBUG'] = 'true' and both hyperresource and oauth2 gems will use excon and display the full logging of the requests and responses